src/Components/Profile/ProfileCreator.tsx

```
@@ -291,6 +291,20 @@ const ProfileCreator = () => {
     },
   });
 
+  const { users, isPending } = GetUserQuery();
+
+  // Check if profile.name is already set
+  useEffect(() => {
+    if (users && users.profile.name && users.profile.name.trim() !== "") {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)